XamarinCommunityToolkitSample/Pages/AttachedProperties/ProgressBarAttachedPropertiesPage.xaml

```diff
@@ -0,0 +1,21 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<pages:BasePage xmlns="http://xamarin.com/schemas/2014/forms"
+                xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+                xmlns:attachedprops="clr-namespace:Xamarin.CommunityToolkit.AttachedProperties;assembly=Xamarin.CommunityToolkit"
+                xmlns:pages="clr-namespace:Xamarin.CommunityToolkit.Sample.Pages"
+                xmlns:vm="clr-namespace:Xamarin.CommunityToolkit.Sample.ViewModels.AttachedProperties"
+                x:Class="Xamarin.CommunityToolkit.Sample.Pages.AttachedProperties.ProgressBarAttachedPropertiesPage">
+    <pages:BasePage.BindingContext>
+        <vm:ProgressBarAttachedPropertiesViewModel />
+    </pages:BasePage.BindingContext>
+
+    <StackLayout Padding="{StaticResource ContentPadding}"
+                 Spacing="50"
+                 VerticalOptions="CenterAndExpand">
+        <ProgressBar attachedprops:ProgressBarAttachedProperties.AnimatedProgress="{Binding Progress}" />
+
+        <Button Text="Set to 0" Command="{Binding SetTo0Command}"  />
+        <Button Text="Set to 50" Command="{Binding SetTo50Command}" />
+        <Button Text="Set to 100" Command="{Binding SetTo100Command}" />
```

Comment:
These copies are not localized.
But actually, I think it's so difficult to maintain localization for the sample project

Reply:
True... should I put it in the RESX file?

Reply:
Good question
I guess yes if we support localization for the entire project so far